Source.NetCore/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+	"supports": {
+		"net46.app": { },
+		"dnxcore50.app": { }
+	},
+	"dependencies": {
+		"Microsoft.NETCore": "5.0.0",
+		"Microsoft.NETCore.Portable.Compatibility": "1.0.0",
+		"NuSpec.ReferenceGenerator": "1.3.6",
+		"System.Diagnostics.StackTrace": "4.0.0",
+		"System.Reflection.Emit": "4.0.0",
+		"System.Reflection.Extensions": "4.0.0",
+		"System.Reflection.TypeExtensions": "4.0.0"
+	},
+	"frameworks": {
+		"dotnet": {
```

Comment:
@Maxwe11 This is the 'old' modern PCL in VS 2015.  The new monikers will most likely require new version of toolset so I haven't updated this.